### PR TITLE
Add clickpress/contao-clickpress-grid

### PR DESCRIPTION
**Pull request by stefansl**

### CODE REVIEW

File: linter/whitelists/de.txt [hunk 1]
```diff
@@ -328,6 +328,7 @@ Responsive
 responsiven
 Rewrite
 Rewriting
+Rocksolid
```
Comment (xchs) [inline linter/whitelists/de.txt]: Actually, it should be enough if "Rocksolid" is on the default whitelist, isn't it?

File: meta/clickpress/contao-clickpress-grid/de.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    keywords:
+        - contao
+        - contao-grid
+        - contao-bundle
+        - spaltenset
+        - subcolumns
+        - grid
+        - gridcss
+        - columns
+        - comlumnset
```
Comment (xchs) [inline meta/clickpress/contao-clickpress-grid/de.yml]:
```suggestion
        - columnset
```

File: meta/clickpress/contao-clickpress-grid/en.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        Big thanks to rocksolid_columns for the code base.
+    keywords:
+        - contao
+        - contao-grid
+        - contao-bundle
+        - subcolumns
+        - grid
+        - gridcss
+        - columns
+        - comlumnset
```
Comment (xchs) [inline meta/clickpress/contao-clickpress-grid/en.yml]:
```suggestion
        - columnset
```

File: linter/whitelists/de.txt (hunk 1 above)
Comment (stefansl) [reply linter/whitelists/de.txt]: yes, it should. But Travis throwed an error and that's why I added it to de.txt as well.

Comment (xchs) [reply linter/whitelists/de.txt]: Ah, okay.

**Comment by xchs** [comment]
Thank you @stefansl

**Comment by stefansl** [comment]
Thank you! @xchs :)

**Pull request merged**